### PR TITLE
[CI:DOCS] Document behavior of --timestamp when only FROM is used

### PR DESCRIPTION
**Pull request by graywolf-at-work**
Fixes: #15171

Signed-off-by: Tomas Volf <tomas.volf@showmax.com>

#### Does this PR introduce a user-facing change?

```release-note
Improve documentation of podman-build --timestamp
```

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: docs/source/markdown/podman-build.1.md [hunk 1]
```diff
@@ -694,6 +694,8 @@ specified and therefore not changed, allowing the image's sha256 hash to remain
 same. All files committed to the layers of the image will be created with the
 timestamp.
 
+If only instruction in a Containerfile is `FROM`, this flag has no effect.
```
Comment (baude) [inline docs/source/markdown/podman-build.1.md:697]: mind adding "the" between "If" and "only"?

**Comment by rhatdan** [comment]
/approve
/lgtm
/hold
thanks @graywolf-at-work

**Review by flouthoc — APPROVED** [review]
/hold cancel

**Comment by mheon** [comment]
Picked up a merge conflict, can you rebase?

**Pull request merged**